stress/common/src/main/java/alluxio/stress/BaseParameters.java

```diff
@@ -23,6 +23,7 @@
 public final class BaseParameters {
   public static final String CLUSTER_FLAG = "--cluster";
   public static final String CLUSTER_LIMIT_FLAG = "--cluster-limit";
+  public static final String CLUSTER_START_DELAY_FLAG = "--cluster-start-delay";
```

Comment:
`CLUSTER_START_DELAY_FLAG` -> `CLUSTER_START_DELAY`

Reply:
@dbw9580

Reply:
Why stripping the flag suffix? The other options have it.

Reply:
I don't have the bible with me but oftentimes when a parameter is `xxx-flag` it suggests. the parameter is a boolean field. This parameter has a value so it's a bit more logical to not have a `-flag` suffix.

Reply:
Yeah I agree with that, but there are already a lot of options that are named `FLAG` but take an argument, e.g. `CLUSTER_LIMIT_FLAG` and `JAVA_OPT_FLAG`.

Reply:
Ok seems there are arguments from both sides now so I'm ok either way. I will recommend removing `_FLAG` suffix from non-boolean parameters, but that should be done in another PR.